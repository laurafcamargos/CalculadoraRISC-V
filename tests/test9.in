3.14
5
+
2.5
7
f
