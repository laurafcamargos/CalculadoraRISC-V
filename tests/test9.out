Digite um número: Erro: valor não é um número inteiro válido
Digite um número: Digite a operação (+, -, *, /, u, f): Digite um número: Erro: valor não é um número inteiro válido
Digite um número: Resultado: 12
Digite a operação (+, -, *, /, u, f): 
